pkg/cloud/azure/azure.go

```diff
@@ -88,11 +88,11 @@ func NewAzureProvider(
 
 	cloudInfo := azure.CloudInfo{
 		SubscriptionID: subscriptionID,
```

Comment:
This needs an extra space:

```suggestion
		SubscriptionID:  subscriptionID,
```